package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -148,124 +163,90 @@
-
-    .. versionadded:: 2.8.0
-    """
-    C3_ALPHABET = np.array(["-", "H", "E"])
-    index = np.argmax(onehot, axis=-1)
-    return C3_ALPHABET[index]
-
-
 class DSSP(AnalysisBase):
     """Assign secondary structure using DSSP algorithm.
 
     Analyze a selection containing a protein and assign secondary structure
-    using the Kabsch-Sander algorithm [Kabsch1983]_. Only a subset of secondary
-    structure categories are implemented:
+    using the Kabsch-Sander algorithm :footcite:p:`Kabsch1983`. Only a subset
+    of secondary structure categories are implemented:
 
-    - 'H' represents a generic helix (alpha-helix, pi-helix or 3-10 helix)
-    - 'E' represents 'extended strand', participating in beta-ladder
-      (parallel or antiparallel)
+    - 'H' represents a generic helix (α-helix, π-helix or 3-<sub>10</sub> helix)
+    - 'E' represents 'extended strand', participating in beta-ladder (parallel
+      or antiparallel)
     - '-' represents unordered part
 
     The implementation was taken from the pydssp package (v. 0.9.0)
     https://github.com/ShintaroMinami/PyDSSP by Shintaro Minami.
 
     Parameters
     ----------
-    u : Universe
-        input universe
+    atoms : Union[Universe, AtomGroup]
+        input Universe or AtomGroup. In both cases, only protein residues will
+        be chosen. Heavy atoms of the protein are then selected by name
+        ("N", "CA", "C", "O O1 OT1"), and hydrogens are selected by name "H".
     guess_hydrogens : bool, optional
-        whether you want to guess hydrogens positions, by default True.
+        whether you want to guess hydrogens positions, by default ``True``.
         Guessing is made assuming perfect 120 degrees for all bonds that N
         atom makes, and a N-H bond length of 1.01 A.
-        If `guess_hydrogens` is False, hydrogen atom positions on N atoms
+        If ``guess_hydrogens`` is False, hydrogen atom positions on N atoms
         will be parsed from the trajectory, except for the "hydrogen" atom
         positions on PRO residues, and an N-terminal residue.
 
     Raises
     ------
     ValueError
-        if `guess_hydrogens` is True but some non-PRO hydrogens are missing.
+        if ``guess_hydrogens`` is True but some non-PRO hydrogens are missing.
 
     Examples
     --------
 
     For example, you can assign secondary structure for a single PDB file:
 
-    .. code-block:: python
-        from MDAnalysis.analysis.dssp import DSSP
-        from MDAnalysisTests.datafiles import PDB
-        import MDAnalysis as mda
-        u = mda.Universe(PDB)
-        run = DSSP(u).run()
-        print("".join(run.results.dssp[0][:20]))
-        # '--EEEEE-----HHHHHHHH'
+    >>> from MDAnalysis.analysis.dssp import DSSP
+    >>> from MDAnalysisTests.datafiles import PDB
+    >>> import MDAnalysis as mda
+    >>> u = mda.Universe(PDB)
+    >>> run = DSSP(u).run()
+    >>> print("".join(run.results.dssp[0][:20]))
+    --EEEEE-----HHHHHHHH
```

Comment:
```suggestion
    --EEEEE-----HHHHHHHH

    (Note that for displaying purposes we only print the first 20 residues
    of frame 0 with ``run.results.dssp[0][:20]`` but one would typically look
    at all residues ```run.results.dssp[0]``.)
```

Needs explanation.

Reply:
Done, thanks.

Reply:
The actual `print()` line was removed?! It should be put back.

Reply:
My bad, thanks